### ISSUE TITLE
Children are cut off when the screen size is too small

### ISSUE THREAD

**Issue by Kaysen98** [darioielardi/flutter_speed_dial#313]
Hello!

When I use a lot of SpeedDialChildren and the screen size is too small the unfolded items are simply cut off.
Can you add something like a Wrap widget or atleast a scroll widget to make them wrap or scrollable?
Otherwise users cannot use some actions because they are cut off.

Thanks in advance!

**Comment by prateekmedia** [comment]
Screenshot?

**Comment by Kaysen98** [comment]
Hey, I cant do a screenshot right now, but Ive got the speed dial with many items in it.
When I expand it, it renders all the items accordingly on top of each other, but there are so many speeddialchildren, that the height of the column hosting the children is just bigger than the screen height. So they just disappear/overflow.

I looked into the source code a little bit and you are rendering the child elements inside a column (or row, in this case a column) - maybe wrapping a scrollview around it would already fix the issue. Or maybe give the user the option to customize the wrapping widget :)

You can easily reproduce the issue if you add many children to the speeddial until it overflows.

Thanks in advance!

**Comment by Kaysen98** [comment]
For testing purposes I wrapped the column into a singlechildscrollview and added a boxconstraint of max-height to the container - that seems to fix the issue.

**Comment by Kaysen98** [comment]
@prateekmedia You've got time to implement that anytime soon? :)

**Comment by prateekmedia** [comment]
Unfortunately this package follows Material design which limits it to current implementation. We cannot add a scroll view just cause we can.

**Comment by scopendo** [comment]
@prateekmedia – it seems odd to say it has to follow the Material Design limitation -- shouldn't that be up to the app developer to decide?  At the moment, a 5th button overflows the top of the screen in landscape.

**Comment by prateekmedia** [comment]
@scopendo yes that should be upto the app developer. But this package is currently in security fixes mode only, so if you are interested in contributing this feature then it would be highly appreciated.
